src/assets/assetdb.h

```diff
@@ -32,6 +37,23 @@ struct CBlockAssetUndo
         READWRITE(fChangedIPFS);
         READWRITE(strIPFS);
         READWRITE(nUnits);
+        if (ser_action.ForRead()) {
+            if (!s.empty() and s.size() >= 1) {
+                int8_t nVersionCheck;
+                ::Unserialize(s, nVersionCheck);
+
+                if (nVersionCheck == ASSET_UNDO_INCLUDES_VERIFIER_STRING) {
+                    READWRITE(fChangedVerifierString);
```

Comment:
I wouldn't expect this line to be here,  just `::Unserialize(...)` on the next line.  What does this do?

Reply:
Fixed this